ext/ffi/ffi.c

```diff
@@ -4359,6 +4359,35 @@ ZEND_METHOD(FFI, string) /* {{{ */
 }
 /* }}} */
 
+ZEND_METHOD(FFI, isNull) /* {{{ */
+{
+	zval *zv;
+	zend_ffi_type *type;
+	void *ptr = NULL;
+
+	ZEND_FFI_VALIDATE_API_RESTRICTION();
+	ZEND_PARSE_PARAMETERS_START(1, 1)
+		Z_PARAM_ZVAL(zv);
```

Comment:
This can use `Z_PARAM_OBJECT_OF_CLASS(zv, zend_ffi_cdata_ce)` instead of the manual check below.